pkg/network/http/http_statkeeper.go

```diff
@@ -0,0 +1,62 @@
+// +build linux_bpf
+
+package http
+
+import (
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+)
+
+type Key struct {
+	SourceIP   util.Address
```

Comment:
It might be worth it indexing the data using the high/low int64 representation (and doing conversions when you fetch the data), but let's profile first :)